### ISSUE TITLE
Is this someone's cruelty project?

### ISSUE THREAD

**Issue by romanr** [civitai/civitai-link-desktop#62]
When you install it there's pass code. Not sure why if there's API key as well.
And on website there's convenient "copy" button to copy the installation passcode. except you CAN'T PASTE IT. Why?

And then you input API key. IF you find it. because "Get api key" just opens home page" and good luck finding it, there's no API key menu.

So aPI key also has convenient "Copy" button. And there's NO PASTE allowed into API key field.
Why? why be so cruel?

And obviously why nobody ever tested it from the beginning? not developer testing with API key already entered. Because if developer tested it once like actual user, entering that api key, then I wouldn't be writing this.

**Comment by JustMaier** [comment]
Hey @romanr did you follow the install guide for Link?
https://youtu.be/EHUjiDgh-MI

**Comment by romanr** [comment]
> Hey @romanr did you follow the install guide for Link? https://youtu.be/EHUjiDgh-MI

Yes, there's no other way to install it. Yes, my computer is plugged into power and turned on.

**Comment by romanr** [comment]
is this project abandoned? last update half year ago and a lot of important issues.
